CONTRIBUTING.md

```diff
@@ -128,10 +128,12 @@ Also don't forget the documentation (reference documentation for features, javad
 
 ## Update the issue with the correct release information:
 
+If the change would result in behaviour in Narayana that is incompatible with the current release stream for Narayana, then the relevant JIRA issue `Fix Version/s` must be set to the `<version+1>.next`. For example if the root pom.xml is currently `<version>7.0.3.Final-SNAPSHOT</version>` then the JIRA issue should have a `Fix Version/s` of `8.next` added.
```

Comment:
Remark if I read this example literally then the fix version would need to be set to `8.0.3.Final-SNAPSHOT` but I think contributors will understand your intention.